src/components/Header.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Flex, Image, Link } from "@hackclub/design-system";
+
+const Base = Flex.extend`
```

Comment:
only define a styled component when you need more options than are available just with default props. replace with this code:

```js
const Base = Flex.withComponent('header')

const Header = () => (
  <Base justify="space-between">
    …
  </Base>
)
```